conversions/octal_to_binary.py

```diff
@@ -0,0 +1,70 @@
+"""
+* Author: Bama Charan Chhandogi (https://github.com/BamaCharanChhandogi)
+* Description: Convert a Octal number to Binary.
+
+References for better understanding:
+https://en.wikipedia.org/wiki/Binary_number
+https://en.wikipedia.org/wiki/Octal
+"""
+
+
+def octal_to_binary(octal_number: str) -> str:
+    """
+    Convert an Octal number to Binary.
+
+    ValueError: Non-octal value was passed to the function
+    >>> octal_to_binary("Av")
+    Traceback (most recent call last):
+        ...
+    ValueError: Non-octal value was passed to the function
+
+    ValueError: Special Character was passed to the function
+    >>> octal_to_binary("@#")
+    Traceback (most recent call last):
+        ...
+    ValueError: Special Character was passed to the function
+
+    ValueError: Empty String was passed to the function
+    >>> octal_to_binary("")
+    Traceback (most recent call last):
+        ...
+    ValueError: Empty String was passed to the function
+
+    ValueError: Octal value was passed to the function
+    >>> octal_to_binary("17")
+    '001111'
+
+    ValueError: Octal value was passed to the function
+    >>> octal_to_binary("7")
+    '111'
+    """
```

Comment:
```suggestion
    """
    Convert an Octal number to Binary.

    >>> octal_to_binary("17")
    '001111'

    >>> octal_to_binary("7")
    '111'

    >>> octal_to_binary("Av")
    Traceback (most recent call last):
        ...
    ValueError: Non-octal value was passed to the function
    
    >>> octal_to_binary("@#")
    Traceback (most recent call last):
        ...
    ValueError: Non-octal value was passed to the function

    >>> octal_to_binary("")
    Traceback (most recent call last):
        ...
    ValueError: Empty String was passed to the function
    """
```
Place the most import doctests at the top (the ones that demonstrate correct output) and edge cases should come after those. Also, there's no need to describe the error when the doctests already shows the error messages.